development/tools/jobs/kyma/kyma_gardener_integration_test.go

```diff
@@ -27,8 +27,32 @@ func TestKymaGardenerAzureIntegrationJobPeriodics(t *testing.T) {
 	tester.AssertThatHasPresets(t, job.JobBase, preset.GardenerAzureIntegration, preset.KymaCLIStable, preset.NightlyGithubIntegration, preset.KymaKeyring, preset.KymaEncriptionKey)
 	tester.AssertThatHasExtraRefs(t, job.JobBase.UtilityConfig, []string{"test-infra", "kyma"})
 	assert.Equal(t, "eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20200206-22eb97a4", job.Spec.Containers[0].Image)
-	assert.Equal(t, []string{"/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/kyma-gardener-integration.sh"}, job.Spec.Containers[0].Command)
+	assert.Equal(t, []string{"/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh"}, job.Spec.Containers[0].Command)
 	tester.AssertThatContainerHasEnv(t, job.Spec.Containers[0], "KYMA_PROJECT_DIR", "/home/prow/go/src/github.com/kyma-project")
 	tester.AssertThatContainerHasEnv(t, job.Spec.Containers[0], "GARDENER_REGION", "westeurope")
 	tester.AssertThatSpecifiesResourceRequests(t, job.JobBase)
 }
+
+func TestKymaGardenerGCPIntegrationJobPeriodics(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/kyma/kyma-integration-gardener-gcp.yaml")
+	// THEN
+	require.NoError(t, err)
+
+	periodics := jobConfig.Periodics
+	assert.Len(t, periodics, 1)
+
+	jobName := "kyma-gardener-azure-integration"
```

Comment:
Should be `kyma-gardener-gcp-integration`

Reply:
Yes, I actually forgot to commit this change on the test...
SHould be fixed now.